scripts/globals/garrison.lua

```diff
@@ -0,0 +1,530 @@
+require("scripts/globals/common");
+require("scripts/globals/settings");
+
+---## TODO:
+--* Force confrontation on npcs inside garrison. npc:addStatusEffectEx(EFFECT_CONFRONTATION,0,1000,0,1800); 0
+--* Prevent Fellows/trusts and disallow garrison during ballista
+--* Add SQL tables for ally npcs to spawn, update script to instantiateMob the new npcs
+--* Ensure SQL flag for confrontation is removed on zone, can update similar to this (allows confrontation to wipe on zoning):
+--        update status_effects set flags = flags | integer_value_for_EFFECTFLAG_ON_ZONE where statusid = integer_value_for_EFFECT_BATTLEFIELD;
+--* Once NPC allies are in change updateClaim to attack NPC's instead of player
+--* Update ALL mobs pos in sql
+--* Update instantiateMob to return newMob
+--* Build out other conquest npc scripts per region OP npc
+
+
+-->>>>UPDATE Mob scripts
+--setDropID(4596)
```

Comment:
Do Garrison mobs drop seals and/or crystals? If not, you're going to have to add a mobmod for that (can't use the no drops one because you have mannequin parts) here:

https://github.com/DarkstarProject/darkstar/blob/9b5036e877bd6a28c291b2b0e895997af2ea113c/src/map/entities/mobentity.cpp#L859

and a couple of lines above that for crystals.

Reply:
Not sure about crystals but I was able to get a seal in retail the other day during testing.